src/promptflow/promptflow/_trace/tracer.py

```diff
@@ -0,0 +1,13 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+
+
+class Tracer:
```

Comment:
may name this thing as exporter, as _core.tracer already using that name

Reply:
as offline sync, removed this class/file as seems we don't need a `Tracer` thing.